Formula/libtrng.rb

```diff
@@ -0,0 +1,34 @@
+class Libtrng < Formula
+  desc "Tina's Random Number Generator Library"
+  homepage "https://www.numbercrunch.de/trng/"
+  url "https://www.numbercrunch.de/trng/trng-4.19.tar.gz"
+  sha256 "36b49961c631ae01d770ff481796c8b280e18c6b6e6b5ca00f2b868533b0492e"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
```

Comment:
Unfortunately, simple `make install` does not work and is not intended to work. Please refer to [installation instructions](https://github.com/rabauke/trng4/blob/master/doc/trng.pdf), page 96.